localstack/ext/java/src/main/java/cloud/localstack/LambdaExecutor.java

```diff
@@ -32,6 +37,8 @@
  */
 public class LambdaExecutor {
 
+	private static Object inputObject;
```

Comment:
Is there a specific reason for keeping the `inputObject` as a static variable here? I'd rather avoid keeping state in class-scoped variables. (In fact, we should try to avoid the use of `static` altogether, but that can be tackled in a future PR).

Can we define `inputObject` as a stack variable in the `main` method instead, and change `setInputObject` to `getInputObject`?

Reply:
I'll see if I can make it better. Did not like the static myself :P